x/upgrade/cli/tx.go

```diff
@@ -0,0 +1,88 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/celestiaorg/celestia-app/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("%s transactions subcommands", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(CmdSignalVersion())
+	cmd.AddCommand(CmdTryUpgrade())
+
+	return cmd
+}
+
+func CmdSignalVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "signal [version]",
+		Short: "Signal a software upgrade for the specified version",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			version, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+
+			addr := clientCtx.GetFromAddress().Bytes()
+			valAddr := sdk.ValAddress(addr)
+
+			msg := types.NewMsgSignalVersion(valAddr, version)
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func CmdTryUpgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "try-upgrade",
+		Short: "Try to perform a software upgrade",
+		Long: `This command will submit a TryUpgrade message to tally all
+the signal votes. If a quorum has been reached, the network will upgrade 
```

Comment:
[Question] What indications should a user look for after executing this command to confirm that a network upgrade is in progress/going to happen? I am referring to this part of the command description "If a quorum has been reached, the network will upgrade to the signalled version at the following height."